tests/utils/test_registry.py

```diff
@@ -45,26 +45,42 @@ def __init__(self, str_value: str, int_value: int):
         self.value = str_value, int_value
         pass
 
+@glob_registry.register(name="dummy_functions",force=True)
+def create_dummy(str_value: str, int_value: int):
```

Comment:
I think this kind of test is better put inside the corresponding test function, instead of in global, to better read the flow of the tests.

Reply:
updated